### ISSUE TITLE
[Migrate] Bug when transforming array of loaders

### ISSUE THREAD

**Issue by okonet** [webpack/webpack-cli#153]
**Do you want to request a *feature* or report a *bug*?**
Bug

**What is the current behavior?**

This config will not be transformed correctly:

```js
module: {
    loaders: [
      {
        test: /\.js?$/,
        loaders: ["react-hot-loader/webpack",'babel-loader?presets[]=react,presets[]=es2015'],
        exclude: /node_modules/
      },
    ]
  },
```

It should, however, be transformed to:

```js
module: {
    loaders: [
      {
        test: /\.js?$/,
        ues: [
          {
            loader: "react-hot-loader/webpack",
          }, {
            loader: 'babel-loader',
            options: {
              presets: ['react', 'es2015']
            }
          }
        ],
        exclude: /node_modules/
      }
    ]
  },
```

**Comment by ev1stensberg** [comment]
On #150 I get this:

![skjermbilde 2017-05-18 kl 12 54 37](https://cloud.githubusercontent.com/assets/16735925/26220936/948b589e-3bc9-11e7-84bb-20e250eb1916.png)

**Comment by okonet** [comment]
Confirming. Will take a look soon.

**Comment by okonet** [comment]
This should be fixed in #150
